scripts/install_from_scratch.sh

```diff
@@ -34,10 +34,10 @@ make test
 # Install environment variables.
 cat << "EOF" > .env
```

Comment:
@akprasad Well, this script is already unpredictable if you run it outside of the root directory, because it writes to the relative path `.env`. So I think this change here should be fine since it is not adding additional unpredictability? (maybe in a future PR we could add the guard you mentioned).

Reply:
Ah, fair point. Let's merge it in! Thanks for the PR.